eng/pipelines/templates/jobs/ci.yml

```diff
@@ -22,6 +22,14 @@ parameters:
   default: []
 - name: TestTimeoutInMinutes
   type: number
+- name: MatrixConfigs
+  type: object
+- name: AdditionalMatrixConfigs
```

Comment:
I don't see this used.

Reply:
It's not used by any ci tests currently, but it's a feature we support for matrix configuration. The same parameter is used by a few live tests, and potentially ci tests in the future now that this is available.